src/test/java/org/mitre/synthea/engine/StateTest.java

```diff
@@ -88,9 +88,10 @@ public void setup() throws IOException {
     person.attributes.put(Person.BIRTHDATE, birthTime);
 
     String testStateDefault = Config.get("test_state.default", "Massachusetts");
-    PayerManager.loadPayers(new Location(testStateDefault, null));
+    PayerManager.loadPayers(new Location(testStateDefault, null),
+        Utilities.getYear(System.currentTimeMillis()));
```

Comment:
I'm a little confused why we're adding an `endYear` parameter all over the place... the rabbit hole seems to end at `PayerManager.csvLineToPlan(...)`... why not just default endYear to `Integer.MAX_VALUE` if the column in the spreadsheet is blank? It would seem to have the same effect, and would eliminate the need for adding that argument everywhere.

Reply:
Agreed, implemented as suggested.